From c487350a3a02c04a63cfb17b87c4800c30c759b7 Mon Sep 17 00:00:00 2001
From: Ron Eldor <Ron.Eldor@arm.com>
Date: Thu, 6 Dec 2018 17:21:52 +0200
Subject: [PATCH 28/43] Add comment describing the feature

Add a comment that describes that the feature only supportes the
`use_srtp` extension, and not hte full DTLS-SRTP RFC.
---
 include/mbedtls/config.h | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/include/mbedtls/config.h b/include/mbedtls/config.h
index f80e8b48a..44c1d0f61 100644
--- a/include/mbedtls/config.h
+++ b/include/mbedtls/config.h
@@ -1571,7 +1571,10 @@
 /**
  * \def MBEDTLS_SSL_DTLS_SRTP
  *
- * Enable support for DTLS-SRTP, RFC5764
+ * Enable support for DTLS-SRTP, RFC5764, use_srtp extension.
+ * \note Only the dtls-srtp key material negotiation is supported.
+ * Once negotiated, the key should be extracted, and data should be transmitted
+ * via an SRTP stack.
  *
  * Requires: MBEDTLS_SSL_PROTO_DTLS
  *
-- 
2.17.1

